app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -6,9 +6,30 @@ class MonthlyKeyMetricsReport < BaseReport
 
     attr_reader :report_date
 
-    def perform(date)
+    def perform(date = Time.zone.today)
       @report_date = date
-      csv_for_email = monthly_key_metrics_report_array
+
+      account_reuse_table = account_reuse_queries.account_reuse_report
+      total_profiles_table = account_reuse_queries.total_identities_report
+
+      upload_to_s3(account_reuse_table, report_name: 'account_reuse')
+      upload_to_s3(total_profiles_table, report_name: 'total_profiles')
+
+      email_tables = [
+        [
+          {
+            title: "IDV app reuse rate #{account_reuse_queries.stats_month}",
+            float_as_percent: true,
+            precision: 4,
+          },
+          account_reuse_table,
+        ],
+        [
+          { title: 'Total proofed identities' },
+          total_profiles_table,
```

Comment:
I believe these need to be splatted so that the header "row" is a row
```suggestion
          *account_reuse_table,
        ],
        [
          { title: 'Total proofed identities' },
          *total_profiles_table,
```